Dockerfile

```
@@ -1,40 +1,18 @@
-FROM ubuntu:20.04
+FROM python:3.11.9
 
-ENV DEBIAN_FRONTEND noninteractive
+ENV PYTHONUNBUFFERED=1 \
+    PYTHONPATH=/opt/sbosc
 
-# apt update
-RUN apt-get update && \
-    apt -y upgrade && \
-    apt-get install -y software-properties-common && \
-    add-apt-repository -y ppa:deadsnakes/ppa && \
-    apt-get update
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    default-libmysqlclient-dev \
```

```suggestion
RUN apt-get update && apt-get install -y --no-install-recommends \
    build-essential \ 
    default-libmysqlclient-dev \
```
`build-essential` was required when `slim` base image was used